Gopkg.toml

```diff
@@ -24,41 +24,37 @@
   name = "github.com/container-storage-interface/spec"
   version = "1.0.0"
 
-[[constraint]]
-  name = "github.com/kubernetes-csi/csi-test"
-  version = "~v1.0.3"
+#[[constraint]]
+#  name = "github.com/kubernetes-csi/csi-test"
+#  version = "~v1.0.3"
 
-[[constraint]]
-  name = "github.com/kubernetes-csi/external-snapshotter"
-  version = "1.0.0-rc4"
+#[[constraint]]
+#  name = "github.com/kubernetes-csi/external-snapshotter"
+#  version = "1.0.0-rc4"
 
-[[constraint]]
+[[override]]
   name = "github.com/kubernetes-sigs/sig-storage-lib-external-provisioner"
```

Comment:
This `name` may also need to switch to `sigs.k8s.io/sig-storage-lib-external-provisioner`. I kept running `dep ensure` failures without the above switch - so just wanted to call it out.